run_tests.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+CMD="coverage run --rcfile=../coverage-config --source='.' manage.py test --settings=concent_api.settings.testing"
+
+display_usage(){
+    printf "Usage:\n run_tests [-p || --pattern <tests pattern>] [-n || --multicore <number of cores to use>]\n"
+}
+
+for i in "$@"
```

Comment:
to enforce parallel run you use option `-n=4` or `--multicore=4`. Have you tried it? It works ;)

Reply:
``` bash
./full-check.sh --multicore 8
```
```
(...)
========================= UNIT TESTS WITH COVERAGE =================
Usage:
 run_tests [-p || --pattern =<tests pattern>] [-n || --multicore =<number of cores to use>]
```

Reply:
What about this one?

Well, I'm going to approve the pull request anyway because it's not something we can't work around and I do not want to keep the pull request open too long but this is really inconsistent. `manage.py` accepts the `--parameter value` form and that's what we're using most of the time in deployment. I can't really think of a command-line utility where this form would not be accepted and equivalent to ``it would not be equivalent with the `--parameter=value` form.

If I were writing this myself I'd just use `$@` and pass all the script parameters down to `manage.py` instead of processing in the bash script.